migra/changes.py

```diff
@@ -3,6 +3,8 @@
 from collections import OrderedDict as od
 from functools import partial
 
+from networkx import lexicographical_topological_sort, DiGraph
```

Comment:
Should this use `TopologicalSorter` which is part of `schemainspect`: https://github.com/djrobstep/schemainspect/blob/066262d6fb4668f874925305a0b7dbb3ac866882/schemainspect/graphlib/__init__.py#L38 ?

Also, since I'm not sure migra/schemainspect are maintained. Would you have any interest opening similar PRs for this and https://github.com/djrobstep/schemainspect/pull/90 against https://github.com/mmkal/pgkit? I ported migra and schemainspect to typescript in that repo, and I intend to continue to maintain them.